docs/util/schema.go

```diff
@@ -0,0 +1,642 @@
+			AdditionalProperties bool   `json:"additionalProperties"`
+			Properties           struct {
+				Name struct {
+					Type string `json:"type"`
+				} `json:"name"`
+				Title struct {
+					Type        string `json:"type"`
+					Description string `json:"description"`
+				} `json:"title"`
+				Image struct {
+					Type        string `json:"type"`
+					Description string `json:"description"`
+				} `json:"image"`
+				Email struct {
+					Type        string `json:"type"`
+					Description string `json:"description"`
+					Format      string `json:"format"`
+				} `json:"email"`
+				Phone struct {
+					Type        string `json:"type"`
+					Description string `json:"description"`
+				} `json:"phone"`
+				Url struct {
+					Type        string `json:"type"`
+					Format      string `json:"format"`
+					Description string `json:"description"`
+				} `json:"url"`
+				Summary struct {
+					Type        string `json:"type"`
+					Description string `json:"description"`
+				} `json:"summary"`
+				CurrentLocation struct {
+					Type        string `json:"type"`
+					Format      string `json:"format"`
+					Description string `json:"description"`
+					Properties  struct {
+						Lat struct {
+							Type string `json:"type"`
+						} `json:"lat"`
+						Long struct {
+							Type string `json:"type"`
+						} `json:"long"`
+					} `json:"properties"`
+				} `json:"currentLocation"`
+				PermanentLocation struct {
+					Type        string `json:"type"`
+					Format      string `json:"format"`
+					Description string `json:"description"`
+					Properties  struct {
+						Lat struct {
+							Type string `json:"type"`
+						} `json:"lat"`
+						Long struct {
+							Type string `json:"type"`
+						} `json:"long"`
+					} `json:"properties"`
+				} `json:"permanentLocation"`
+			} `json:"properties"`
+		} `json:"core"`
+		Work struct {
+			Type            string `json:"type"`
+			AdditionalItems bool   `json:"additionalItems"`
+			Items           struct {
+				Type                 string `json:"type"`
+				AdditionalProperties bool   `json:"additionalProperties"`
+				Properties           struct {
+					Name struct {
+						Type        string `json:"type"`
+						Description string `json:"description"`
+					} `json:"name"`
+					Description struct {
+						Type        string `json:"type"`
+						Description string `json:"description"`
+					} `json:"description"`
+					Position struct {
+						Type        string `json:"type"`
+						Description string `json:"description"`
+					} `json:"position"`
+					Location struct {
+						Type        string `json:"type"`
+						Format      string `json:"format"`
+						Description string `json:"description"`
+						Properties  struct {
+							Lat struct {
+								Type string `json:"type"`
+							} `json:"lat"`
+							Long struct {
+								Type string `json:"type"`
+							} `json:"long"`
+						} `json:"properties"`
+					} `json:"location"`
+					Url struct {
```

Comment:
For some reason GitHub marks my comments as resolved so I'm writing them again:

Why switch from `URL` to `Url` ? I belive the idiom in Go is to capitalize 3 letter acronyms

Reply:
@alarsyo On the template file (doc.tmpl) the url is generated as "Url" which makes unable to parse this "URL"

Reply:
That's probably because we use `strings.Title` :/ 
I'm not sure what's best in this case, stick to the Go idiom and add a special case like 
```go
if var == "url" {
        var = strings.ToUpper(var)
}
```
or just keep it simple like this even if we don't stick to best naming practices. Probably just leave it as is, allows to have a much simpler code.

Reply:
I would agree with leave it as is for now. In the schema itself we are using lowercase "url", so switching to URL seems a move away from the json schema + complicating code.